src/common/routes.ts

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import IRoute from "./interface";
+import * as pageRoutes from "./pageRoutes";
+//Lazy routes
+const HomePage = React.lazy(() => import("../Home/Home"));
```

Comment:
I prefer absolute paths over relative ones. It makes it easier to move files/modules around in a predictable fashion.

Can you change these and add the same to the documentation.